tests/dashbio_demos/app_alignment_viewer.py

```diff
@@ -1,10 +1,11 @@
+import json
+
```

Comment:
Actually you want the blank line after `import os` (block of standard library imports).